### PR TITLE
Remove support for wildcard account ID

### PR DESCRIPTION
**Pull request by jodosha**

### CODE REVIEW

File: content/v2.markdown [hunk 1]
```diff
@@ -169,7 +169,7 @@ The order of fields is relevant, as it will determine the priority of the sortin
 
 ### Example
 
-List all domains for the account 1010 sorting them by expiration date (asc) and id (desc)
+List all domains from the account `1010` sorting them by expiration date (asc) and id (desc)
```
Comment (weppos) [inline content/v2.markdown]: Is _from_ correct here?

/cc @aeden @jcaudle @arnamak

Comment (jcaudle) [inline content/v2.markdown]: "from" is probably better than "for", but "in" seems like it'd be best.

**Comment by jodosha** [comment]
@jcaudle Thank you 👍 
@weppos Do you think we can merge it now?

**Comment by weppos** [comment]
👍

**Pull request merged**